src/Model/User.php

```diff
@@ -1113,7 +1113,7 @@ public static function getStatistics()
 			FROM `user`
 			INNER JOIN `profile` ON `profile`.`uid` = `user`.`uid`
 			INNER JOIN `contact` ON `contact`.`uid` = `user`.`uid` AND `contact`.`self`
-			WHERE (`profile`.`publish` OR `profile`.`net-publish`) AND `user`.`verified`
+			WHERE `user`.`verified` AND `user`.`login_date` > '0001-01-01' AND NOT `user`.`account-type` = 3
```

Comment:
What about the other non-person account types, should they be excluded as well:
- Organisation (maybe?)
- News (probably)
- Relay (definitely)

Reply:
Hmm. I'd rather not exclude any account type but someone suggested to exclude forums. So I excluded them in this draft. The purpose of the changed query is to include all valid accounts to get accurate numbers.

What is a 'relay' account? I really have no idea. :-)

Reply:
I have little idea what a relay account is, but I'm pretty sure it doesn't fit into the general idea of "user count".

Reply:
I checked the description at the top of the file...

> ACCOUNT_TYPE_RELAY - the account is a relay
> This will only be assigned to contacts, not to user accounts

So, I think this in not relevant as we only count accounts not contacts.

Reply:
I don't know. This relay thing is confusing. Maybe I should include forums and exclude those relay "accounts". What do you think?

Reply:
From what you found out it seems like you don't have to do it since it isn't relevant to users, only contacts, and you're counting users.

Reply:
I don't think, that _organizations_ should be excluded from the counting.  For most parts they are just as normal interacting with other users as any other account.